### PR TITLE
Takeover advantage

### PR DESCRIPTION
**Pull request by deadlight**
## Done

A continuation of work partially reviewed in: #1143 

New homepage takeover and swapped-out takeunder

## QA

Look at the homepage takeover and the right-hand takeunder at all viewports.

### CODE REVIEW

File: templates/takeovers/_advantage.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<section class="row row-hero no-border-bottom strip-dark no-border">
```
Comment (anthonydillon) [inline templates/takeovers/_advantage.html]: Duplicate `no-border` class.

File: templates/takeovers/_advantage.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <div class="strip-inner-wrapper equal-height">
+        <div class="six-col append-one equal-height__item equal-height__align-vertically">
+            <h1>Secure your production environment with Ubuntu Advantage</h1>
+            <div class="twelve-col for-small">
+                <img style="margin:2rem auto;display:block;" src="{{ ASSET_SERVER_URL }}191e226e-infographic_Ubuntu-Advantage_svg.svg" alt="" class="row-hero__image" />
+            </div>
+            <p>Need the assurance of 24/7 world class support for your Ubuntu deployment? We&apos;ve got you covered.</p>
+            <p><a class="button--primary row-hero__cta" href="https://buy.ubuntu.com/?utm_campaign=Advantage&utm_medium=takeover&utm_source=ubuntuwebsite"><span class="external">Explore Ubuntu Advantage Store</span></a></p>
```
Comment (anthonydillon) [inline templates/takeovers/_advantage.html]: lowercase Store?

File: templates/takeovers/_advantage.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            <p><a style="color:#fff" href="/support">Find out more&nbsp;›</a></p>
```
Comment (anthonydillon) [inline templates/takeovers/_advantage.html]: Inline styles?

**Comment by anthonydillon** [comment]
Could you also look into this issue please:
the leading operating system for pcs tablets phones iot devices servers and the cloud ubuntu

File: templates/takeovers/_advantage.html (hunk 3 above)
Comment (deadlight) [inline templates/takeovers/_advantage.html]: yeah, I don't know why that was in there, removed.

**Pull request merged**